tools/build.sh

```diff
@@ -1,8 +1,8 @@
 #!/bin/sh
-set -e
+set -x -e
```

Comment:
It's nice to have some indication of where that script got to, should it fail.  Given the amount of output it already  produces when it succeeds, printing a little more seems acceptable.